ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNTAyNjI3OTEzLjE3NDcxMzI3NTgifSwiYW5vbklkIjoiNzY1ZTE3ZGU2ODYxMDkzNiJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5wdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zdXYvZ2xjL292ZXJ2aWV3Lmh0bWw%2FaW50ZXJuYWxfdGVzdD10cnVlIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MiIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjIzMjA1NDY2NDQ4MzA4OTk4IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x55941d8a27fa <unknown>
#1 0x55941d346e90 <unknown>
#2 0x55941d32e156 <unknown>
#3 0x55941d32ca6a <unknown>
#4 0x55941d32d1cf <unknown>
#5 0x55941d32d124 <unknown>
#6 0x55941d34b11f <unknown>
#7 0x55941d3f03ca <unknown>
#8 0x55941d3be242 <unknown>
#9 0x55941d3e4135 <unknown>
#10 0x55941d3be013 <unknown>
#11 0x55941d38ab3b <unknown>
#12 0x55941d38b7a1 <unknown>
#13 0x55941d867b9b <unknown>
#14 0x55941d86ba8a <unknown>
#15 0x55941d84f912 <unknown>
#16 0x55941d86c604 <unknown>
#17 0x55941d83474f <unknown>
#18 0x55941d890678 <unknown>
#19 0x55941d890856 <unknown>
#20 0x55941d8a1666 <unknown>
#21 0x7fa25769caa4 <unknown>
#22 0x7fa257729c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyNTQ2IiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJwdF9QVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnB0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbD9pbnRlcm5hbF90ZXN0PXRydWUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYyIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI1MDI2Mjc5MTMuMTc0NzEzMjc1OCJ9LCJhbm9uSWQiOiI3NjVlMTdkZTY4NjEwOTM2In0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyNTQ2IiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyNTQ2Il19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMzQyNDYxNTE4NDg3MzQ1MSJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x55941d8a27fa <unknown>
#1 0x55941d346e90 <unknown>
#2 0x55941d32e156 <unknown>
#3 0x55941d32ca6a <unknown>
#4 0x55941d32d1cf <unknown>
#5 0x55941d32d124 <unknown>
#6 0x55941d34b11f <unknown>
#7 0x55941d3f03ca <unknown>
#8 0x55941d3be242 <unknown>
#9 0x55941d3e4135 <unknown>
#10 0x55941d3be013 <unknown>
#11 0x55941d38ab3b <unknown>
#12 0x55941d38b7a1 <unknown>
#13 0x55941d867b9b <unknown>
#14 0x55941d86ba8a <unknown>
#15 0x55941d84f912 <unknown>
#16 0x55941d86c604 <unknown>
#17 0x55941d83474f <unknown>
#18 0x55941d890678 <unknown>
#19 0x55941d890856 <unknown>
#20 0x55941d8a1666 <unknown>
#21 0x7fa25769caa4 <unknown>
#22 0x7fa257729c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjUwMjYyNzkxMy4xNzQ3MTMyNzU4In0sImFub25JZCI6Ijc2NWUxN2RlNjg2MTA5MzYifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoucHQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sP2ludGVybmFsX3Rlc3Q9dHJ1ZSIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjIiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI3ODc5NzUyNjkxMTY3NzcxIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=136.0.7103.59)
Stacktrace:
#0 0x55941d8a27fa <unknown>
#1 0x55941d346e90 <unknown>
#2 0x55941d32e156 <unknown>
#3 0x55941d32ca6a <unknown>
#4 0x55941d32d1cf <unknown>
#5 0x55941d32d124 <unknown>
#6 0x55941d34b11f <unknown>
#7 0x55941d3f03ca <unknown>
#8 0x55941d3be242 <unknown>
#9 0x55941d3e4135 <unknown>
#10 0x55941d3be013 <unknown>
#11 0x55941d38ab3b <unknown>
#12 0x55941d38b7a1 <unknown>
#13 0x55941d867b9b <unknown>
#14 0x55941d86ba8a <unknown>
#15 0x55941d84f912 <unknown>
#16 0x55941d86c604 <unknown>
#17 0x55941d83474f <unknown>
#18 0x55941d890678 <unknown>
#19 0x55941d890856 <unknown>
#20 0x55941d8a1666 <unknown>
#21 0x7fa25769caa4 <unknown>
#22 0x7fa257729c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:19 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'